lambda/write-user-services/index.ts

```diff
@@ -43,6 +43,7 @@ export const lambdaHandler = async (event: SQSEvent): Promise<void> => {
     try {
       const userServices = parseRecordBody(event.Records[i].body);
       validateUserServices(userServices);
+      /* eslint no-await-in-loop: "off" */
```

Comment:
Would it make more sense to turn it off globally [a config file](https://eslint.org/docs/latest/user-guide/configuring/rules#using-configuration-files)?

I guess it depends if it really is just this case, or if we think this is common enough we'll start seeing this comment turn up around?

I'll approve as non-blocking, but Ideally I reckon we should prefer rules files over inline comments where possible?

Reply:
I agree that we should block rules in the config if we never want them to apply, but I don't think that's the case here.

The only place I expect this skip command to appear is in the try / catch block in the top level functions. Everywhere else in the lambda we *should* be avoiding `await` in loops, so I decided to keep the rule enabled to cover those.